crates/api/src/doc/glossary.rs

```diff
@@ -0,0 +1,9 @@
+//! A glossary of terms used in the Kitsune2 project.
+//!
+//! ## Time slice
+//! A time slice is a period of time. It is defined to be a an interval from a start time,
+//! inclusive, up to some end time, exclusive.
```

Comment:
```suggestion
//! inclusive, up to some end time, exclusive. It can be called a time partition as well.
```

Reply:
I will go back through my other doc comments and be more careful about this. A partition isn't intended to refer to an individual slice within the partition, the partition is meant to refer to the collection of time slices

Reply:
Ah, okay. And a space partition is then the collection of space slices? Does that sound right?

Reply:
We need a term for this. I wasn't going to use slice. I've been using segment but that's actually wrong and what I should be saying is "sector"... Dear me, evolving new terminology is messy. I'm just going to have to keep going over these docs and making them consistent with themselves

Reply:
Yeah, sector is the geometrical term for a space partition with its time slices. I like the circle analogy. We could also use a line with stacks on it. Anyhow, off this goes.